### ISSUE TITLE
Wrong inaka_mixer dependency

### ISSUE THREAD

**Issue by joaohf** [inaka/cowboy-swagger#51]
My rebar.config is:

```
{deps, [
      {cowboy_swagger, "0.1.1"}
]}.
``

```
joaohf@porco:~/p/x$ make
===> Verifying dependencies...
===> Package inaka_mixer-0.1.4 not found. Fetching registry updates and trying again...
===> Updating package registry...
===> Writing registry to /home/joaohf/.cache/rebar3/hex/default/registry
===> Generating package index...
===> Writing index to /home/joaohf/.cache/rebar3/hex/default/packages.idx
===> Package not found in registry: inaka_mixer-0.1.4.
Makefile:10: recipe for target 'compile' failed
make: *** [compile] Error 1 
```

$ rebar3 hex search swagger
cowboy_swagger: 0.1.1
$ rebar3 hex search mixer
inaka_mixer: 0.1.5

**Comment by elbrujohalcon** [comment]
@joaohf that issue seems to have been fixed. Can you try with the latest release (i.e. [1.0.1](https://github.com/inaka/cowboy-swagger/releases/tag/1.0.1)), please?

**Comment by elbrujohalcon** [comment]
@joaohf I'll assume it's fixed on 1.0.1
